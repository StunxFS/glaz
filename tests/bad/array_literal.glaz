fn main() {
	_ = vec[i32]{}; // invalid
	_ = [i32; _]::new(); // invalid

	_ = [i32; 3]::new(1, 2, 3); // invalid
}
