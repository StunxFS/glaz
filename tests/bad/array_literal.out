tests/bad/array_literal.glaz:2:7: error: invalid vector literal
     | 
   2 | 	_ = vec[i32]{}; // invalid
     | 	    ^^^^^^^^^^
     = note: 1 or more elements are expected
     = help: use `vec[T]::new()` instead
tests/bad/array_literal.glaz:3:7: error: unable to calculate array size
     | 
   3 | 	_ = [i32; _]::new(); // invalid
     | 	    ^^^^^^^^^^^^^^^
     = help: add a size: `[T; 5]::new()
tests/bad/array_literal.glaz:5:7: error: array literal do not take arguments
     | 
   5 | 	_ = [i32; 3]::new(1, 2, 3); // invalid
     | 	    ^
glaz error: could not compile `array_literal`, aborting due to 3 previous errors.
